spec/registry/unicode.md

```diff
@@ -0,0 +1,63 @@
+# MessageFormat 2.0 Unicode Registry
+
+The `u:` namespace is reserved for use by the Unicode Consortium.
+
+## Options
+
+This section describes common options which each implementation SHOULD support
+for all _functions_ and _markup_.
```

Comment:
Perhaps:

```suggestion
This section describes _options_ which are intended to be common to all _functions_.
Implementations SHOULD support resolving and passing these _options_.
_Function_ authors SHOULD implement each _option_ as described here to ensure interoperability.
```

Reply:
The proposed language doesn't quite match the [accepted design](https://github.com/unicode-org/message-format-wg/blob/main/exploration/expression-attributes.md#contextual-options), though. These are not intended to be options that are handled by each function, but by the implementation, and they e.g. apply changes to the function context, so that function authors explicitly do not need to do anything to enable their use.